modules/datablocksBidAdapter.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+```
+Module Name: Datablocks Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: support@datablocks.net
+```
+
+# Description
+
+Connects to Datablocks Version 5 Platform
+Banner Native and 
+
+
+# Test Parameters
+```
+    var adUnits = [
+      {
+        code: 'banner-div',
+        sizes: [[300, 250]],
+        mediaType:{
+        	banner: {
+        		sizes: [300,250]
+        	}
+        },
+        bids: [
+          {
+            bidder: 'datablocks',
+            params: {
+              sourceId: 12345,
+              host: 'prebid.datablocks.net'
+            }
+          }
+        ]
+      }, {
+        code: 'native-div',
+        bids: [
+          {
+            bidder: 'datablocks',
+            mediaType : {
```

Comment:
Minor note - this field should be named `mediaTypes` and it should be located next to the `bids` property, not inside it.

Can you update the example here?

Reply:
Done in latest commit. Thanks!